src/Microsoft.DotNet.XHarness.InstrumentationBase.Xunit/Microsoft.DotNet.XHarness.InstrumentationBase.Xunit.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <TargetFrameworks>netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <RootNamespace>Microsoft.DotNet.XHarness.TestRunners.Xunit</RootNamespace>
+    <OutputType>Library</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.DotNet.XHarness.TestRunners.Xunit\Microsoft.DotNet.XHarness.TestRunners.Xunit.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Mono.Android">
+      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\MonoAndroid\v11.0\Mono.Android.dll</HintPath>
```

Comment:
As discussed on the sync, this looks fishy and won't work for non-Windows platforms. Can we get this from a NuGet maybe?

Reply:
It seems that there is no suitable NuGet for this. Since I use `Instrumentation` and `Bundle` from there I should rework the change significantly. Let me discuss this with @mattleibow

Reply:
Yeah, there must be a way since Matthew also must be implementing it somehow in the place that you are trying to now replace

Reply:
Think think it is because I am building an Android library. Basically using msbuild.sdk.extras with a tfm of monoandroid10.0

Reply:
You should be able to multi target across netstd and monoandroid.

Reply:
Removed this dependency and targeted monoandroid

Reply:
It is not preserved after update